packages/api/src/errors.js

```diff
@@ -117,6 +117,24 @@ export class MagicTokenRequiredError extends HTTPError {
 }
 MagicTokenRequiredError.CODE = 'ERROR_MAGIC_TOKEN_REQUIRED'
 
+export class TosAgreementRequiredError extends HTTPError {
+  constructor (msg = 'Missing Terms of Service agreement') {
+    super(msg, 400)
+    this.name = 'TosAgreementRequired'
+    this.code = TosAgreementRequiredError.CODE
+  }
+}
+TosAgreementRequiredError.CODE = 'ERROR_TOS_AGREEMENT_REQUIRED'
```

Comment:
No need to change. I see this is how a lot of the rest of the file does it. But for my edification, is this different than including `static CODE = 'ERROR_TOS_AGREEMENT_REQUIRED'` in the class definition? I don't think so but could be wrong.

Reply:
It's the same.  I also felt it was an odd way of writing it, but it follows the existing pattern.